### ISSUE TITLE
[tech] Speed up the retirement receipt page load time

### ISSUE THREAD

**Issue by Atmosfearful** [KlimaDAO/klimadao#764]
After a user creates a carbon offset retirement, and clicks the link to their retirement page, there is a few seconds of delay. ?Let's try to improve this.

* [x] prefetching can help
```html
<link
    rel="prefetch"
    href={`https://klimadao.finance/retirements/${data.beneficiary_address}/${data.retirement_index}`}
/>
```
* [x] add a subgraph poller so we don't try navigating / prefetching the page until we know the record exists in subgraph

**Comment by Atmosfearful** [comment]
Related to #1018

**Comment by jabby09** [comment]
@Atmosfearful I added this to the /retire epic

**Comment by psparacino** [comment]
addressed in: https://github.com/KlimaDAO/klimadao/pull/1331

**Comment by psparacino** [comment]
This issue was addressed in PR #1331
